feedstock/meta.yaml

```diff
@@ -13,14 +13,12 @@ provenance:
       roles:
```

Comment:
Not sure exactly how this relates.
`"Primary research institution developing the neural network and coordinating data integration"`